server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -53,9 +77,21 @@ exports.createLetters = async (req, res) => {
 
 exports.updateLetters = async (req, res) => {
     try {
+        const {id} = req.params
+        const {quantity} = req.body
 
-    } catch (e) {
+        const letter = await Letters.findById(id)
+
+        if(!letter){
+            return res.status(404).json({ error:'Letter not found'})
+        }
+
+        letter.totalStock += quantity
```

Comment:
Yeah, it felt weird adding that in there, so I guess that's why - because I assumed it would be adding OR taking away from that number

Reply:
This may actually still work if we send a negative value in quantity, but totalstock is totalstock, the number available would have to be incremented or decremented, and only if we have something damage or no longer available should total stock be decremented.

Reply:
Also, I'd like to make mention that `letter.totalStock` doesn't exist. It would be `letter.letters[index].totalStock`. Mongo and mongoose have a special way of handling this. We need to be able to target the specific `sym` property for the symbol and update that objects `totalStock` and `numberAvailable` properties.